### PR TITLE
feat: seo improvements for pagination

### LINKED ISSUE #1608: [SEO Improvements] Pagination
## Feature description
- Convert `Button` to anchors with `$as="a"` potentially

### PR DESCRIPTION
**Pull request by jh3y**
#### Description
- Fixes #1608 

#### Scope

- [ ] Patch: Bug Fix
- [x] Minor: New Feature
- [ ] Major: Breaking Change

### CODE REVIEW

**Comment by gergelyke** [comment]
can we keep the original look and feel?

Screen Shot 2019-07-25 at 9 46 57 AM

**Comment by jh3y** [comment]
Maybe we should close this one. It's to do with the `disabled` attribute. I didn't notice in the scenario that we are using `disabled` on the `Button` dependent on the `currentPage`.

The alternative is to style the anchor tag based on the `currentPage` and conditionally fire the `onPrevClick` handler. We don't have styling in place for a `disabled` button though so it would be a case of trying to match it up.

**Comment by gergelyke** [comment]
> Maybe we should close this one. It's to do with the `disabled` attribute. I didn't notice in the scenario that we are using `disabled` on the `Button` dependent on the `currentPage`.
> 
> The alternative is to style the anchor tag based on the `currentPage` and conditionally fire the `onPrevClick` handler. We don't have styling in place for a `disabled` button though so it would be a case of trying to match it up.

Sounds good - I am okay with closing this. I am following up with the SEO consultant because some of the recommendations seem really hard to follow.

**Comment by jh3y** [comment]
Thanks 👍
